bom/camel-bom/pom.xml

```diff
@@ -566,6 +566,11 @@
         <artifactId>camel-dataset</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.camel</groupId>
+        <artifactId>camel-datasonnet</artifactId>
+        <version>${project.version}</version>
+      </dependency>
```

Comment:
I don't think these should have been merged. I think the cache in GH is outdated causing regen bot to re-add these.